tests/integration/helpers/cluster.py

```diff
@@ -2056,6 +2056,17 @@ def restart_instance_with_ip_change(self, node, new_ip):
     def restart_service(self, service_name):
         run_and_check(self.base_cmd + ["restart", service_name])
 
+    def restart_container(self, container_name):
```

Comment:
Perhaps existing function "run_and_check" will do the job already? (as helpers/cluster.py is generic framework code, I'd be reluctant with adding too much stuff that is called only from a single place)